config/vmdb.tmpl.yml

```diff
@@ -238,6 +238,9 @@ smtp:
 snapshots:
   :create_free_percent: 100
   :remove_free_percent: 100
+ssl:
+  :ssl_ca_file:
+  :ssl_ca_path:
```

Comment:
oh ... right ... refresh vs. event catcher.

If @mcornea says one root cert, then that's probably closer to the truth.

Ignore my comment about moving it ... we can always revisit this.